docs/getting-started/accessing-resources/getting-access-to-data.md

```diff
@@ -120,3 +130,95 @@ This will download test data and direct `data/current` symlink to the test data
 
 To switch back to using ScPCA data, rerun the script with the `--release current` option.
 If you already have the most recent data, this will not repeat downloading the data you already have.
+
+
+## Accessing ScPCA module results
+
+Once an analysis modules matures, the Data Lab ports the analysis to a separate repository called [`OpenScPCA-nf`](https://github.com/AlexsLemonade/OpenScPCA-nf).
+This repository holds a [Nextflow workflow](https://www.nextflow.io/) to reproducibility run and generate results from completed analysis modules.
+
+If you wish to use output from a completed analysis module in your module, you will need to obtain the results generated by this workflow.
+
+We have provided a [`download-results.py` script](https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/main/download-results.py) that you can use to download all results from a module that has been run in `OpenScPCA-nf`.
+Results are available at specific tagged releases. <!-- STUB_LINK to openscpca-nf docs -->
```

Comment:
The results bucket is organized by data release, not workflow release, so I think the tagging comment here is potentially confusing?

Reply:
I just went ahead and removed it.